jetty-http/src/main/java/org/eclipse/jetty/http/HttpParser.java

```diff
@@ -1945,15 +1939,15 @@ public default void badMessage(BadMessageException failure)
          * @param reason the response reason phrase
          * @return true if handling parsing should return
          */
-        public boolean startResponse(HttpVersion version, int status, String reason);
+        boolean startResponse(HttpVersion version, int status, String reason);
     }
 
     /* ------------------------------------------------------------------------------- */
     /* ------------------------------------------------------------------------------- */
     /* ------------------------------------------------------------------------------- */
     public interface ComplianceHandler extends HttpHandler
```

Comment:
Move this out into a top level interface, it's no longer specific to HttpParser.